Add notebooks folder and update classification tutorial from notebook (#119)
scikit-learn-contrib/MAPIE: notebooks/README.md

> This folder contains a series of notebooks used for generating the tutorials for regression and classification as well as some heavier notebooks which cannot be saved in the MAPIE documentation.

Reviewer: ```suggestion
This folder contains a series of notebooks used for generating the tutorials for regression and classification as well as some other notebooks not represented in MAPIE examples.
```

Author: Done !
